# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/ipc/PWindowGlobal.ipdl
# Commit: 7229926a45c2
# Full Hash: 7229926a45c2e43136c7f8f597a92776658edbd5
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2020-10-14 09:52:12
# Description:
#   Bug 1660539 - Make StructuredCloneData args optional in JSActor RawMessage, r=kmag
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D93122
# ==============================================================================

diff -r 1aa2dc4b280e -r 7229926a45c2 dom/ipc/PWindowGlobal.ipdl
--- a/dom/ipc/PWindowGlobal.ipdl	Tue Oct 13 16:57:32 2020 +0000
+++ b/dom/ipc/PWindowGlobal.ipdl	Tue Oct 13 17:43:56 2020 +0000
@@ -78,8 +78,8 @@
   async AddBlockedFrameNodeByClassifier(MaybeDiscardedBrowsingContext aNode);
 
 both:
-  async RawMessage(JSActorMessageMeta aMetadata, ClonedMessageData aData,
-                   ClonedMessageData aStack);
+  async RawMessage(JSActorMessageMeta aMetadata, ClonedMessageData? aData,
+                   ClonedMessageData? aStack);
 
 parent:
   // Load the given URI load state into the current owner process of the given